### ISSUE TITLE
Compilation of Pokémon Sword and Pokémon Shield issues

### ISSUE THREAD

**Issue by MysticExile** [yuzu-emu/yuzu#3320]
Per the request of a developer, I have compiled all current open issues that concern Pokémon Sword [0100abf008968000] and Pokémon Shield [01008db008c2c000].

The game soft locks.
=============
This happens for example when you finish the first cutscene and the player character is sitting on the couch. Or when you encounter the game's legendary Pokémon in Slumbering Weald.

Wireframe on certain models.
===================
Certain NPCs have a wireframe mesh across them.
This affects most NPCs and the Player character, but it does not affect and Pokémon.
![69242376-a2a6db00-0b98-11ea-8bc4-1b3098f7dd85](https://user-images.githubusercontent.com/30736337/72665834-7975c000-3a0c-11ea-9ee7-e81cbe922a43.png)
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/4080837/yuzu_log.txt)

Rotomb Box screen does not render as it should.
===============================
yuzu Mainline 61
![68995819-3b381500-0892-11ea-987d-3e803c700f62](https://user-images.githubusercontent.com/30736337/72665891-1a647b00-3a0d-11ea-9d21-25ef79378a12.png)
Nintendo Switch
![68995822-5571f300-0892-11ea-851f-d1b48325c2d4](https://user-images.githubusercontent.com/30736337/72665895-2cdeb480-3a0d-11ea-9ede-4505f9ca6ebc.png)

Flickering in several areas of the game.
=========================
In several areas of the game, the shadows will violently flicker like this. It can be replicated by standing in this exact spot/area in Hulbery (Though it also occurs in many other places)

The same bug occurs on OpenGL & Vulkan, with and without Async GPU.
![71546568-89176100-2998-11ea-8e6a-5a72233243bd](https://user-images.githubusercontent.com/30736337/72665971-d3c35080-3a0d-11ea-906e-75cfdb067aad.gif)

Running in Tall Grass is not possible.
========================
It is currently not possible to run/sprint in any tall/long grass in the game (While it is possible on the Nintendo Switch). The player character simply defaults to the slow-moving/sneaking animation, regardless of input.

Video Example off the problem: https://youtu.be/16obz1-pwBM

Minor graphical issues.
===============
The shading on the character is incorrect as you can see in the examples below.
yuzu
![69053742-c213fb80-0a12-11ea-8acf-68950f0a6270](https://user-images.githubusercontent.com/30736337/72666297-f99e2480-3a10-11ea-9179-7fe767c3682e.png)
Nintendo Switch
![69012548-bcb9a100-097f-11ea-99d5-9f6accc2395c](https://user-images.githubusercontent.com/30736337/72666301-04f15000-3a11-11ea-910a-40b8dfd4698e.png)

Time shows incorrectly.
===============
Time always shows 1/1/1970
This will be fixed with #3271

**Comment by swablueme** [comment]
Although dates seem to advance normally in SwSh, this doesn't seem to impact upon events within the game. For example, in max raids, the pokemon/power of dens always seem to remain the same day by day and won't naturally reroll every single day.

**Comment by zakion1** [comment]
![image](https://user-images.githubusercontent.com/61523047/75394173-98257b80-5915-11ea-80ed-fe0766b6bfec.png)
 the weather doesn't change in wild area in yuzu as it completely ignoring the pc mentioned time,that means certain pokemon can't be encountered in wild area if the game doesn't resets the weather to change there pokemon area spawn , it all works in regular switch

**Comment by Hexagon12** [comment]
Softlocks and tall grass bugs will be fixed with #3725

**Comment by TheOnlyPhantom** [comment]
> Although dates/times now seem to advance normally in SwSh (for example, when saving the game), this doesn't seem to impact upon events within the game. For example, in max raids, the pokemon/power of dens always seem to remain the same day by day and won't naturally reroll every single day like they would on a regular switch. Pokemon jobs also won't progress.

I experienced this as well. It seems to be resetting timers on timed events similar to how the game does on the console if you change the system time or date. But it happens in yuzu if you shut the emulation session down at all. I have confirmed that leaving the emulation session running for the duration of a timed event allows it to progress.

**Comment by MohamedAarab** [comment]
> > Although dates/times now seem to advance normally in SwSh (for example, when saving the game), this doesn't seem to impact upon events within the game. For example, in max raids, the pokemon/power of dens always seem to remain the same day by day and won't naturally reroll every single day like they would on a regular switch. Pokemon jobs also won't progress.
> 
> I experienced this as well. It seems to be resetting timers on timed events similar to how the game does on the console if you change the system time or date. But it happens in yuzu if you shut the emulation session down at all. I have confirmed that leaving the emulation session running for the duration of a timed event allows it to progress.

Is there a way to advance time in-game really fast instead of keeping the emulation session running for 24 hours?

**Comment by MBTheGamingMaster** [comment]
The Rotom Box layout is now fixed. Kindly update the issue description.

**Comment by MysticExile** [comment]
@MBTheGamingMaster Thanks! Can you point to the PR that fixed this?

**Comment by MBTheGamingMaster** [comment]
> @MBTheGamingMaster Thanks! Can you point to the PR that fixed this?

Sorry but I don't recall the exact PR that fixed that.

**Comment by MarioSonic2987** [comment]
Setting GPU Accuracy level to Extreme, it fixes dark shading on models.

**Normal/High**:
![image](https://user-images.githubusercontent.com/20027681/88304438-90250880-cd08-11ea-8e2e-9b75c451f3ed.png)
**Extreme**:
![image](https://user-images.githubusercontent.com/20027681/88304443-931ff900-cd08-11ea-8905-192323992bd0.png)

And there's stuttering particles effects when GPU Accuracy level is set to Normal, like in Ballonlea. You need to use High level to fix it:
![image](https://user-images.githubusercontent.com/20027681/88304955-34a74a80-cd09-11ea-9938-38f524082764.png)

**Comment by Serfrost** [comment]
Regarding [ The shading on the character is incorrect ... ] Please note that it affects all 3D entities, not just the player character. - this can temporarily be "resolved" by using Vulkan and then setting the GPU Accuracy to Extreme. OpenGL crashes on my end when trying to use the Extreme option.

After setting it to Extreme, run through some of the shaded problem-areas and then, afterwards, set the GPU Accuracy back to Normal or High. Magically the problem doesn't return, at least for awhile / in that area of the map.  

My best guess is whatever handles the game's shadow shading just never proceeds to fade away the first time; once you cause that to happen, it seems you're fine. The alternative is that the game considers all areas to be under shade at first, until you force it to recognize that it isn't.  

This is an annoying bandaid fix and I wish there was a hotkey to toggle Extreme mode on and off fairly quickly.  Navigating into the menu each time just takes all that much extra time and annoyance; otherwise I'd be fine with a mod that removed the shading entirely until it's fixed.

**Comment by DAREDEVIL1070** [comment]
It shows error 0100abf008968000 and the game doesn't open

**Comment by RiedleroD** [comment]
ping! - all issues are fixed by now. Just tested it, to be safe. Oh and the wireframe issue was said to be fixed in one of the progress reports, so idk how this got missed ngl.

**Comment by yemaowuxin** [comment]
> ping! - all issues are fixed by now. Just tested it, to be safe. Oh and the wireframe issue was said to be fixed in one of the progress reports, so idk how this got missed ngl.

Weather doesn't resets in yuzu when the dates are changed.
That's not resolved yet.

**Comment by lKobi** [comment]
I agree @yemaowuxin time sensitive games like Pokémon are just ruined, we are forced to cheat the system and use custom RTC, but even then the problem is not entirely fixed.

yuzu is triggering the Pokémon Sword/Shield anti-cheat, which causes custom RTC issues too.

**RiedleroD** has no saying if all the issues are fixed or not anyway, and the developers are aware of the RTC issues, so just be patient.

**Comment by tornadotm** [comment]
Game soft-crashes directly after there are 2 pokemon thrown during the fight with Milo, frames go to "nan ms" and the music loops.
